commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/CPSearchResultsDisplayContext.java

```diff
@@ -318,7 +318,7 @@ protected String getURLString() {
 		_cpContentListEntryRendererRegistry;
 	private final CPContentListRendererRegistry _cpContentListRendererRegistry;
 	private final CPDefinitionHelper _cpDefinitionHelper;
-	private final CPRequestHelper _cpRequestHelper;
+	private final CPSearchResultsRequestHelper _cpRequestHelper;
```

Comment:
`CPSearchResultsRequestHelper _cpRequestHelper` should be `CPSearchResultsRequestHelper _cpSearchResultsRequestHelper`